projects/ng-tailwind/src/components/ngt-dropzone/ngt-dropzone.component.ts

```diff
@@ -140,6 +141,27 @@ export class NgtDropzoneComponent extends NgtBaseNgModel implements OnInit, OnDe
         this.destroySubscriptions();
     }
 
+    public onClick(element, fileType: 'image' | 'archive') {
+        this.disableClick = true;
+
+        if (fileType == 'image') {
+            let ngtDropzoneComponent = this;
+
+            let viewer = new Viewer(element, {
+                ...this.imageViewerOptions, ...{
+                    hidden() {
+                        ngtDropzoneComponent.disableClick = false;
```

Comment:
Não daria para acessar o this.disableClick diretamente e remover essa variável?

Reply:
Acontece com o this nesse caso de perder a referência da classe